### PR TITLE
Register-Button

### PR DESCRIPTION
**Pull request by Jvil26**

### CODE REVIEW

**Review by farisashai — CHANGES_REQUESTED** [review]
Branch Preview
Screenshot 2022-11-06 at 4 12 46 PM

Live Website
Screenshot 2022-11-06 at 4 12 55 PM

The register button looks good so far but seems like the alignment on the rest of the hero text section got messed up and pushed down a lot further by some of the CSS in this PR. Can we try to minimize the changes in `src/sections/Hero/style.module.scss` to only code we need for the register button?

Basically undo these changes and see if that looks better - Also merge the most recent changes on main into this branch to double check the integration

Screenshot 2022-11-06 at 4 14 59 PM

**Review by farisashai — APPROVED** [review]
Made some minor tweaks but register button is good to go
